log/sequencer.go

```diff
@@ -11,13 +11,15 @@ import (
 	"github.com/google/trillian/merkle"
 	"github.com/google/trillian/storage"
 	"github.com/google/trillian/util"
+	"golang.org/x/net/context"
 )
 
-// Sequencer instances are responsible for integrating new leaves into a log.
+// Sequencer instances are responsible for integrating new leaves into a single log.
 // Leaves will be assigned unique sequence numbers when they are processed.
 // There is no strong ordering guarantee but in general entries will be processed
 // in order of submission to the log.
 type Sequencer struct {
+	ctx        context.Context
```

Comment:
Feels a bit weird to store this here, would it make sense to pass it in as a parameter to `SequenceBatch()` and pass it around?